pkgs/development/python-modules/analytics-python/default.nix

```
@@ -0,0 +1,52 @@
+{ lib
+, buildPythonPackage
+, fetchPypi
+, pythonOlder
+, requests
+, monotonic
+, backoff
+, python-dateutil
+}:
+
+buildPythonPackage rec {
+  pname = "analytics-python";
```

```suggestion
  pname = "segment-analytics-python";
```
attribute name and pname should match the name of PyPI.
https://github.com/NixOS/nixpkgs/blob/master/doc/languages-frameworks/python.section.md#contributing-guidelines-contributing-guidelines

---

Ok didn't know the package got renamed. I guess you wouldn't want the old one merged either way so I will check if this will work with nixified-ai.